solutions/observability/apm/resource-attributes.md

```diff
@@ -36,13 +40,119 @@ processors:
 ...
 ```
 
-::::{tip}
-Need to add event attributes instead? Use attributes—​not to be confused with resource attributes—​to add data to span, log, or metric events. Attributes can be added as a part of the OpenTelemetry instrumentation process or with the [attributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor).
+## Handling of unmapped attributes
+
+Only a subset of OpenTelemetry resource attributes are directly mapped to ECS fields. If an attribute doesn't have a predefined ECS mapping, it's stored under `labels.*`, with `.` (dots) replaced by `_` (underscores).
+
+The following table shows how OTel resource attributes are mapped to ECS fields and how they're stored if unmapped.
+
+| OpenTelemetry attribute | Mapped ECS field | If unmapped, stored as |
+|------------------------|------------------|------------------------|
+| `service.name` | `service.name` | - |
+| `service.version` | `service.version` | - |
+| `deployment.environment` | `service.environment` | - |
+| `cloud.provider` | `cloud.provider` | - |
+| `cloud.account.id` | `cloud.account.id` | - |
+| `otel.library.name` | ❌ Not mapped | `labels.otel_library_name` |
+| `custom.attribute.with.dots` | ❌ Not mapped | `labels.custom_attribute_with_dots` |
```

Comment:
At first glance it looks like there's some specific rules around `otel.library.name` and `custom.attribute.with.dots` as it is not called out as an example.

Reply:
Rewriting